.gitignore

```diff
@@ -11,4 +11,4 @@
 __pycache__/
 .cache
 *.pyc
-mlops-stacks.iml
+mlops-stacks.iml
```

Comment:
nit newline

Reply:
I added and removed  a file in my local .gitignore, the changes are just the space removed after .iml
Should we add *.iml in the file or do you want me revert the changes (adding the space back) ?

Reply:
Let's revert the changes